<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1156</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QToolButton" name="toolButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>海贼王</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>:/res/QQ截图20210124104205.png</normaloff>:/res/QQ截图20210124104205.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonTextBesideIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>登录</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>66</width>
      <height>84</height>
     </rect>
    </property>
    <property name="title">
     <string>性别</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="radioButtonMan">
       <property name="text">
        <string>男</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButtonWomen">
       <property name="text">
        <string>女</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>81</width>
      <height>84</height>
     </rect>
    </property>
    <property name="title">
     <string>婚否</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_3">
       <property name="text">
        <string>已婚</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>未婚</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>91</width>
      <height>136</height>
     </rect>
    </property>
    <property name="title">
     <string>GroupBox</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>11111</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_2">
       <property name="text">
        <string>22222</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_3">
       <property name="text">
        <string>33333</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox_4">
       <property name="text">
        <string>44444</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>281</width>
      <height>271</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string> 锄禾日当午</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </item>
   </widget>
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>10</y>
      <width>311</width>
      <height>271</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string notr="true">1</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>0</y>
      <width>401</width>
      <height>311</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>姓名</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>性别</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>年龄</string>
     </property>
    </column>
    <item row="0" column="0">
     <property name="text">
      <string>一</string>
     </property>
    </item>
    <item row="0" column="1">
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item row="0" column="2">
     <property name="text">
      <string>13</string>
     </property>
    </item>
    <item row="1" column="0">
     <property name="text">
      <string>二</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>16</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>三</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>女</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>18 </string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>四</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>男</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>33</string>
     </property>
    </item>
    <item row="4" column="0">
     <property name="text">
      <string>五</string>
     </property>
    </item>
    <item row="4" column="1">
     <property name="text">
      <string>女</string>
     </property>
    </item>
    <item row="4" column="2">
     <property name="text">
      <string>14</string>
     </property>
    </item>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>300</y>
      <width>301</width>
      <height>271</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>181</width>
        <height>231</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>158</width>
         <height>435</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pushButton_7">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_6">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_5">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_8">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_16">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_15">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_13">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_14">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_9">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_10">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_12">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_11">
          <property name="text">
           <string>PushButton</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QToolBox" name="toolBox">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>85</width>
        <height>171</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="page_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>85</width>
         <height>111</height>
        </rect>
       </property>
       <attribute name="label">
        <string>家人</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="page_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>85</width>
         <height>111</height>
        </rect>
       </property>
       <attribute name="label">
        <string>朋友</string>
       </attribute>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="page_5">
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>261</width>
        <height>221</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string> 百度</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>谷歌</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>  搜狗</string>
       </attribute>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>117</width>
      <height>137</height>
     </rect>
    </property>
    <property name="title">
     <string>切换页面</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="btn_scrollArea">
       <property name="text">
        <string>scrollArea</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_toolBox">
       <property name="text">
        <string>toolBox</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_tabWidget">
       <property name="text">
        <string>tabWidget</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>340</y>
      <width>87</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>奔驰</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>宝马</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>拖拉机</string>
     </property>
    </item>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>550</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>550</y>
      <width>46</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
